clients/drcachesim/tests/basic_counts.templatex

```diff
@@ -32,6 +32,7 @@ Total counts:
 #else
           13 total transfer markers
 #endif
+     .* total function markers
```

Comment:
A slightly more specific test would be nice. Something like a simplified version of static_burst.cpp, where the app calls malloc, say, exactly 1000 times between dr_app_start() and dr_app_stop_and_cleanup(). Then we can run the basic_counts tool same way as here and check we've recorded them.

Reply:
Sure! Will add one in next commit

Reply:
Done